sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.tutorials.Ibor
+{
+    /// <summary>
+    /// Orders represent an instruction from an investor to buy or sell a quantity of a specific
+    /// security.
+    /// </summary>
+    [TestFixture]
+    public class Orders
+    {
+        private ILusidApiFactory _apiFactory;
+        private InstrumentLoader _instrumentLoader;
+        private IOrdersApi _ordersApi;
+        private IList<string> _instrumentIds;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _apiFactory = LusidApiFactoryBuilder.Build("secrets.json");
+            _instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = _instrumentLoader.LoadInstruments();
+            _ordersApi = _apiFactory.Api<IOrdersApi>();
+        }
+
+        [Test]
+        public void Upsert_Simple_Order()
+        {
+            var testScope = $"TestScope-{Guid.NewGuid().ToString()}";
+            var orderId = $"Order-{Guid.NewGuid().ToString()}";
+            
+            // We want to make a request for a single order. The internal security id will be mapped on upsert
+            // from the instrument identifiers passed.
+            var request = new OrderSetRequest(
+                orderRequests: new List<OrderRequest>
+                {
+                    new OrderRequest(
+                        code: orderId,
+                        quantity: 100,
+                        instrumentIdentifiers: new Dictionary<string, string>()
+                        {
+                            ["Instrument/default/LusidInstrumentId"] = _instrumentIds.First()
```

Comment:
Presumably these have the same resolution behaviours as transactions? i.e. they don't have to be defined as unique identifiers in LUISD?

Reply:
Not quite sure what you mean - they're resolved to a Lusid internal instrument on upsert, so they have to exist as instruments in LUSID (or we'll resolve to unknown)?

Reply:
Sounds like its the same. With transactions you can specify a map of identifierType:indentifierValue where the identifierValue can be of a none unique type and so long as it resolves to a single LUID all is well otherwise it resolves to ZZZ